packages/now-cli/src/util/config/global-path.ts

```diff
@@ -15,12 +31,18 @@ const getNowDir = () => {
   });
 
   const customPath = args['global-config'];
+  const xdgConfigPaths = XDGAppPaths('now').dataDirs();
+  const possibleConfigPaths = [homeConfigPath, ...xdgConfigPaths];
 
-  if (!customPath) {
-    return path.join(homedir(), '.now');
-  }
-
-  return path.resolve(customPath);
+  // customPath is the preferred location
+  // the legacy home directory config path is the second most wanted location
+  // otherwise the first matching xdg-config-path is used which already exists
+  // at last the first best xdg-config-path is used
+  return (
+    (customPath && path.resolve(customPath)) ||
+    possibleConfigPaths.filter(configPath => isDirectory(configPath))[0] ||
```

Comment:
```suggestion
    possibleConfigPaths.find(configPath => isDirectory(configPath)) ||
```

I guess we can just use the first one that is a directory

Reply:
Good point, I somehow always don't remember the `find`-function. Thank you.